### PR TITLE
Review

### PR DESCRIPTION
**Pull request by hervenivon**
@ChristopheBougere Good job!

Ça va être sympa.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -389,7 +391,7 @@ <h3>Retry/Catch</h3>
         </section>
         <section>
           <section>
-            <h2>A few example of what we've built</h2>
+            <h2>A simple example of what we've built</h2>
```
Comment (ChristopheBougere) [inline index.html]: Should be plurals actually (5 sub-sections).

File: index.html [hunk 2]
```diff
@@ -463,21 +465,23 @@ <h3>Local execution</h3>
...
           <ul>
-            <li>Easy way to monitor executions (aggregate all the CloudWatch streams)</li>
-            <li>Access the execution ID (add a context object)</li>
-            <li>Dynamic parallel branches</li>
-            <li>Provide a tmp storage available from all the execution</li>
-            <li>Directly trigger executions from API Gateway, SNS, S3 or CloudWatch triggers, ...</li>
+            <li>Better monitoring (aggregate CloudWatch streams)</li>
+            <li>Access to the execution ID (add a context object)</li>
+            <li>Dynamic and non-waiting parallel branches</li>
+            <li>Greate event size</li>
```
Comment (ChristopheBougere) [inline index.html]: Greater?

**Review by ChristopheBougere — COMMENTED** [review]
Thanks :)

**Comment by hervenivon** [comment]
Nickel

**Pull request merged**